packages/dataviews/src/dataviews-layouts/grid/index.tsx

```diff
@@ -139,6 +143,11 @@ function GridItem< Item >( {
 					);
 				}
 			} }
+			role={ isInfiniteScroll ? 'article' : undefined }
```

Comment:
Interesting. I think I'll need a different role for the data picker changes. I'm not sure whether `article` supports selection. Will need to figure out if infinite scroll can work for a picker or if that's an invalid combination.

I'll also need `posinset` so it's useful to establish it here.

Reply:
> Does it makes sense to have view config options in a picker scenario?

Hard to say, possibly a smaller subset of options. But maybe it's not worth differentiating in that case and instead providing all the options.